chemprop/v2/cli/train.py

```diff
@@ -254,22 +259,21 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
         "--task-type",
         default="regression",
         action=LookupAction(ReadoutRegistry),
-        help="Type of dataset. This determines the default loss function used during training.",
+        help="Type of task. This determines the default loss function used during training.",
     )
     data_args.add_argument(
         "--spectra-phase-mask-path",
         help="Path to a file containing a phase mask array, used for excluding particular regions in spectra predictions.",
     )
-    data_args.add_argument(
-        "--data-weights-path",
-        help="a plaintext file that is parallel to the input data file and contains a single float per line that corresponds to the weight of the respective input weight during training. v1 help message: Path to weights for each molecule in the training data, affecting the relative weight of molecules in the loss function.",
-    )
+    # data_args.add_argument(
+    #     "--data-weights-path",
+    #     help="a plaintext file that is parallel to the input data file and contains a single float per line that corresponds to the weight of the respective input weight during training. v1 help message: Path to weights for each molecule in the training data, affecting the relative weight of molecules in the loss function.",
+    # )
```

Comment:
With `--weight-column`, I think we can remove this block instead of just commenting it out, since they are providing the same information

Reply:
just need to make a note that this argument has been removed and subsumed into the input data frame